### PR TITLE
Extra line break in `molecule status` for better visibility

### PR DESCRIPTION
**Pull request by abrown-sg**
Before:
screen shot 2016-02-17 at 10 42 35 am

After:
screen shot 2016-02-17 at 10 42 43 am

### CODE REVIEW

File: molecule/commands.py [hunk 1]
```diff
@@ -436,6 +436,7 @@ def execute(self):
         print(x)
         print
         self.molecule._print_valid_platforms()
+        print
```
Comment (rgreinho) [inline molecule/commands.py]: Just to be consistent, can we use the print function ``print()`` instead of the keyword. Same goes for line 437.

**Comment by rgreinho** [comment]
:+1: after my remarks.

**Comment by abrown-sg** [comment]
@rgreinho Cleaned up the use of print keyword for this change and the last. Interestingly, `print()` will be treated like the keyword and will literally print '()' to the screen. Had to print a blank string to get the intended behavior.

**Comment by rgreinho** [comment]
@abrown-sg Good to know. It is a python 2 specific behavior though. Maybe a python bug?

```
Python 3.5.1 (default, Dec  7 2015, 21:59:08)
[GCC 4.2.1 Compatible Apple LLVM 7.0.0 (clang-700.1.76)] on darwin
Type "help", "copyright", "credits" or "license" for more information.
>>> print()

>>>
```

**Comment by abrown-sg** [comment]
@rgreinho Yeah, looks like a behavior of 2.X.  Should we use print_function from __future__?

```
>>> print()
()
>>> from __future__ import print_function
>>> print()

```

**Comment by rgreinho** [comment]
Sure, that does not sound like a bad idea to me.

**Comment by abrown-sg** [comment]
@rgreinho @retr0h I went ahead and cleaned up the usage of print() everywhere and standardized on using the __future__ version. We were already doing this in utilities. May as well make it consistent.  __future__ imports have to come first in the file.

**Comment by rgreinho** [comment]
Big :+1: !

**Comment by retr0h** [comment]
Why don't we just have `utilities.print()` that implements future print vs having to import all over the place?

**Comment by rgreinho** [comment]
Hummm... because you did not propose this great idea before? :)

**Comment by abrown-sg** [comment]
@retr0h @rgreinho It doesn't look like you can redefine the print statement as a function in Python 2 because it's a language construct. Attempting to do `def print(line):...` in utilities ends in tears. I think to do what @retr0h suggested we'd have to name it something else. Not sure I prefer utilities.my_print() everywhere over just living with the import and using print(). Or doing away with the import and keeping the 2.x behavior until we make a real push toward 3.x compatibility.

**Comment by abrown-sg** [comment]
Disregard that, looks like it's possible with the __future__ stuff. That said, is using utilities.print() everywhere instead of print() cleaner and more clear than a single import line?

**Comment by rgreinho** [comment]
Right, it is kind of the the same. I'm fine with the ``__future__`` import.

**Comment by abrown-sg** [comment]
@rgreinho @retr0h Threw another quick fix into this since it's related.

**Comment by rgreinho** [comment]
:+1:

**Pull request merged**